diff --git a/Sources/Purchasing/StoreKitAbstractions/SubscriptionPeriod.swift b/Sources/Purchasing/StoreKitAbstractions/SubscriptionPeriod.swift
index 63ed98454..22479dfcf 100644
--- a/Sources/Purchasing/StoreKitAbstractions/SubscriptionPeriod.swift
+++ b/Sources/Purchasing/StoreKitAbstractions/SubscriptionPeriod.swift
@@ -35,7 +35,7 @@ public final class SubscriptionPeriod: NSObject, Codable {
 
     /// Units of time used to describe subscription periods.
     @objc(RCSubscriptionPeriodUnit)
-    public enum Unit: Int {
+    public enum Unit: Int, Codable {
 
         /// A subscription period unit of a day.
         case day = 0
@@ -270,5 +270,3 @@ fileprivate extension SubscriptionPeriod.Unit {
 }
 
 // MARK: - Encodable
-
-extension SubscriptionPeriod.Unit: Codable { }
